### PR TITLE
AP-313 Adapt design for History/Favorite entries

### PR DESCRIPTION
**Pull request by jayeshsolanki93**

### CODE REVIEW

File: app/src/lumen/res/layout/history_viewholder.xml [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center">
+
+    <RelativeLayout
```
Comment (spacifici) [inline app/src/lumen/res/layout/history_viewholder.xml]: Avoid `RelativeLayout`

Comment (jayeshsolanki93) [reply app/src/lumen/res/layout/history_viewholder.xml]: I copied this file from cliqz. And here it looks like we need the children to be arranged in relation to one another.

Comment (spacifici) [reply app/src/lumen/res/layout/history_viewholder.xml]: I did it myself

**Comment by jayeshsolanki93** [comment]
Minor issue: the rows do not look exactly the same as in cliqz. 'time' text is below 'description' text in lumen
<p float="left">
 
 
</p>

**Pull request merged**